Zilon.Core/Zilon.Core/Tactics/Spatial/IMapNodeDistanceCalculator.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace Zilon.Core.Tactics.Spatial
+﻿using Zilon.Core.Graphs;
+
+namespace Zilon.Core.Tactics.Spatial
 {
-    public interface IMapNodeDistanceCalculator<TNodeMap> where TNodeMap: IMapNode
+    public interface IMapNodeDistanceCalculator<TNodeMap> where TNodeMap: IGraphNode
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'in' keyword to parameter 'TNodeMap' to make it 'contravariant'.](https://app.codacy.com/app/kreghek/Zilon_Roguelike/pullRequest?prid=4592476)